openedx_learning/apps/authoring/backup_restore/toml.py

```diff
@@ -40,6 +40,8 @@ def toml_learning_package(learning_package: LearningPackage) -> str:
 
 def _get_toml_publishable_entity_table(
         entity: PublishableEntity,
+        draft_version: Optional[PublishableEntityVersion],
+        published_version: Optional[PublishableEntityVersion],
```

Comment:
With more recent Python versions, you can just say:

```suggestion
        draft_version: PublishableEntityVersion | None,
        published_version: PublishableEntityVersion | None,
```

Reply:
Done. thanks